tests/MudBlazor.Markdown.Tests/MarkdownComponentTests/MarkdownComponentShould.cs

```diff
@@ -16,14 +16,14 @@ public void RenderNothingIfNullOrWhitespace(string? value)
 	}
 
 	[Fact]
-	public void RenderCodeItalicAndBold()
+	public void RenderCodeItalicBoldAndStrike()
```

Comment:
Could you please rename the test method name?  
※ If we keep adding more elements to this test, the name length will increase.

`RenderCodeItalicBoldAndStrike` -> `RenderEmphasisElements`